alpenhorn_test.go

```diff
@@ -27,6 +26,7 @@ import (
 	"vuvuzela.io/alpenhorn/pkg"
 	"vuvuzela.io/crypto/rand"
 	"vuvuzela.io/internal/debug"
+	sharedMock "vuvuzela.io/internal/mock"
```

Comment:
Unfortunate that we have two packages with the same name now. We can fix that later.

Reply:
Yeah...